src/js/components/RangeInput/RangeInput.js

```diff
@@ -1,40 +1,119 @@
 import React, { forwardRef, useContext, useState } from 'react';
-
+import { Box } from '../Box';
+import { TextInput } from '../TextInput';
 import { FormContext } from '../Form/FormContext';
 import { StyledRangeInput } from './StyledRangeInput';
 
+const allowedKeysToOpenInputValue = [
+  'ArrowRight',
+  'ArrowLeft',
+  'ArrowUp',
+  'ArrowDown',
+];
+const allowedKeysToCloseInputValue = ['Tab', 'Escape'];
+
 const RangeInput = forwardRef(
   (
-    { a11yTitle, name, onChange, onFocus, onBlur, value: valueProp, ...rest },
+    {
+      a11yTitle,
+      name,
+      onChange,
+      onFocus,
+      onBlur,
+      onKeyDown,
+      min = 0,
```

Comment:
I dont think these should be hardcoded as 0,100,1 
If someone wants other values I don't think this will work properly